ci/scripts/prepare-cached-bitstream.sh

```diff
@@ -19,6 +19,9 @@ BITSTREAM=HEAD ci/bazelisk.sh build --define bitstream=gcp_splice \
   //hw/bitstream:rom_mmi \
   //hw/bitstream:otp_mmi
 
+# Move the bitstream spliced with locally built ROM to the location expected by subsequent steps.
+mv "$(ci/scripts/target-location.sh -b --define -b bitstream=gcp_splice //hw/bitstream:rom)" "$(ci/scripts/target-location.sh //hw/bitstream:rom)"
+
 mkdir -p ${TOPLEVEL_BIN_DIR}
 cp -rLvt "${TOPLEVEL_BIN_DIR}" \
   "$(ci/scripts/target-location.sh //hw/bitstream:test_rom)" \
```

Comment:
wont the rest of these still miss their spliced sources if they depend on --define bitstream=gcp_splice

Reply:
Yes, this was a quick fix to see if we correctly understood the root cause. Last update should clean this up. PTAL